connectors/migrations/db/migration_31.sql

```diff
@@ -0,0 +1,14 @@
+-- Migration created on Nov 07, 2024
+ALTER TABLE "zendesk_tickets" ALTER COLUMN "brandId" DROP NOT NULL;ALTER TABLE "zendesk_tickets" ALTER COLUMN "brandId" DROP DEFAULT;
```

Comment:
OOC: don't we always have a brandId? I feel like other things might break if we don't always have one

Reply:
I think you missed this one @albandum